image/rhel/fetch-stackrox-data.sh

```diff
@@ -4,7 +4,11 @@
 
 set -euxo pipefail
 
+TARGET_DIR="${1:-/stackrox-data}"
+
 fetch_stackrox_data() {
+    local target_dir="${1}"
```

Comment:
[nit] `TARGET_DIR` and `target_dir` mean the same, maybe the former is just redundant since it's used in only one place? I suggest to call `fetch_stackrox_data "${1}"` instead of `fetch_stackrox_data "${TARGET_DIR}"`.

Reply:
Done.